src/Components/Aspire.Microsoft.Azure.Cosmos/AspireAzureCosmosDBExtensions.cs

```diff
@@ -92,6 +107,19 @@ public static class AspireAzureCosmosDBExtensions
             });
         }
 
+        // If the user opts into ignoring the emualtor certificate and the emulator is detected
+        // via the use of the hard coded account key, then disable certificate checking for the
+        // Cosmos DB client.
+        if (settings.IgnoreEmulatorCertificate && IsEmulatorConnectionString(settings.ConnectionString))
```

Comment:
Do we need to do something similar for https://github.com/dotnet/aspire/tree/main/src/Components/Aspire.Microsoft.EntityFrameworkCore.Cosmos?

Reply:
Yes we do ... thanks for picking this up. I'll share some code between them and wire it up.